public/app/core/components/AccessControl/Permissions.tsx

```diff
@@ -20,14 +20,15 @@ const INITIAL_DESCRIPTION: Description = {
 };
 
 export type Props = {
+  title?: string;
   resource: string;
   resourceId: number;
 
   canListUsers: boolean;
   canSetPermissions: boolean;
 };
 
-export const Permissions = ({ resource, resourceId, canListUsers, canSetPermissions }: Props) => {
+export const Permissions = ({ title, resource, resourceId, canListUsers, canSetPermissions }: Props) => {
```

Comment:
```suggestion
export const Permissions = ({ resource, resourceId, canListUsers, canSetPermissions, title='Permissions' }: Props) => {
```
You can set a default title instead, which will be used if no `title` prop is provided.